### PR TITLE
Added documentation to make it easier for first time users to get sta…

### PR DESCRIPTION
**Pull request by fkberthold**
Currently the documentation on the front page is useful if you're already familiar with the project. This is to help people who want to test it on their own servers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ Pore is a plugin for the up-and-coming SpongeAPI built to provide compatibility
 Please note that this project is still under heavy development and **is not yet considered functional**. Currently, no
 Bukkit plugins are verified to be supported.
 
+## Running Pore on Your Sponge Server
+
+If you like living on the bleeding edge and want to give Pore a try:
+
+#. Back up your existing system.
```
Comment (caseif) [inline README.md]: These `#`s need to be dashes. `#` denotes a top-level header.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ Pore is a plugin for the up-and-coming SpongeAPI built to provide compatibility
...
+#. Install [Sponge](https://www.spongepowered.org/), detailed instructions can be found [here](https://docs.spongepowered.org/en/server/getting-started/) .
+#. Copy the [Pore Snapshot](https://ci.caseif.net/job/Pore/lastSuccessfulBuild/artifact/build/libs/Pore-1.0.0-SNAPSHOT.jar) into your server's `mods` directory.
+#. Make a `bukkit-plugins` directory in your server's base directory.
+#. Copy any of the plugins you want to try into the `bukkit-plugins` directory.
+
+When you spin up your server look for messages like:
+
+   [STDERR/sponge]: [name.richardson.james.bukkit.utilities.plugin.AbstractPlugin:onEnable:118]: org.apache.commons.lang3.NotImplementedException: TODO
```
Comment (caseif) [inline README.md]: This should be wrapped in code tags. e.g.:

\`\`\`
code goes here
\`\`\`

File: README.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ Pore is a plugin for the up-and-coming SpongeAPI built to provide compatibility
...
+   
+These indicate that a feature needed for the plugin you're loading hasn't been implemented yet.
```
Comment (caseif) [inline README.md]: May want to indicate that this isn't necessarily a bug (we get a lot of issues and replies reporting these exceptions).

File: README.md [hunk 4]
```diff
@@ -4,6 +4,22 @@ Pore is a plugin for the up-and-coming SpongeAPI built to provide compatibility
 Please note that this project is still under heavy development and **is not yet considered functional**. Currently, no
 Bukkit plugins are verified to be supported.
 
+## Running Pore on Your Sponge Server
+
+If you like living on the bleeding edge and want to give Pore a try:
+
+#. Back up your existing system.
+#. Install [Sponge](https://www.spongepowered.org/), detailed instructions can be found [here](https://docs.spongepowered.org/en/server/getting-started/) .
+#. Copy the [Pore Snapshot](https://ci.caseif.net/job/Pore/lastSuccessfulBuild/artifact/build/libs/Pore-1.0.0-SNAPSHOT.jar) into your server's `mods` directory.
```
Comment (caseif) [inline README.md]: I'd recommend linking to https://ci.caseif.net/job/Pore/lastSuccessfulBuild/ instead of a direct link.

**Comment by caseif** [comment]
Looks good for the most part, though I've left a few comments on minor things that need fixing. :)

**Comment by fkberthold** [comment]
Thanks for the feedback, I've put in fixes. I'm used to using a slightly different dialect of Markdown (pandoc), so used features that don't exist in Github's.

**Comment by caseif** [comment]
Closed via 7b6e647, thanks!

**Pull request closed without merging**